pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.notnoop.apns</groupId>
     <artifactId>apns</artifactId>
-    <version>1.0.0.Beta5-SNAPSHOT</version>
+    <version>1.0.0.bruffos.Beta5</version>
```

Comment:
I will revert that.
/P
On Mon, Jan 12, 2015 at 11:02 AM, Matthias Wessendorf <
notifications@github.com> wrote:
>
> In pom.xml:
>
> > @@ -2,7 +2,7 @@
> >      <modelVersion>4.0.0</modelVersion>
> >      <groupId>com.notnoop.apns</groupId>
> >      <artifactId>apns</artifactId>
> > -    <version>1.0.0.Beta5-SNAPSHOT</version>
> > +    <version>1.0.0.bruffos.Beta5</version>
>
> can you revert these changes ? and the other ones on the piom.xml ?
>
> —
> Reply to this email directly or view it on GitHub.